neurom/io/hdf5.py

```diff
@@ -184,29 +184,29 @@ def remove_duplicate_points(points, groups):
         '''
 
         group_initial_ids = groups[:, 0]
+        group_len = len(group_initial_ids)
 
         def _find_last_point(group_id):
             ''' Identifies and returns the id of the last point of a group'''
+            return group_initial_ids[group_id + 1] - 1
```

Comment:
OK. Yes, I am aware of the ordering of IDs, but wasn't aware if they follow an ordering of sections. Here is an example of what would not work for the previous function:

sec_initial_segment_id, type, parent:
1, 2, 0 -> sec1
100, 2, 3 -> sec2
10, 2, 1 -> sec3

in this case the previous function will fail to compute the last segment id of sec1 correctly. It would return 99 instead of 9.